libgraph/include/katana/GraphTopology.h

```diff
@@ -202,19 +202,19 @@ class KATANA_EXPORT EdgeShuffleTopology : public GraphTopology {
   EdgeShuffleTopology& operator=(const EdgeShuffleTopology&) = delete;
 
   bool is_transposed() const noexcept {
-    return has_transpose_state(tsuba::RDGTopology::TransposeKind::kYes);
+    return has_transpose_state(katana::RDGTopology::TransposeKind::kYes);
```

Comment:
Now that we're all one big happy namespace family we could just remove these rather than replacing them. Not the end of the world to be more verbose though.